Geometry_Engine/Query/ControlPoints.cs

```diff
@@ -72,14 +72,6 @@ public static List<Point> ControlPoints(this Line curve)
 
         /***************************************************/
 
-        [NotImplemented]
```

Comment:
Wonder if we should just remove the ` [NotImplemented]` here rather than wipe the method? I mean, it is correct?

Is if we want controlpoints in a query sense to always be on the curve, but, for nurbs I am not sure if that makes sense?

Reply:
Whoops. I must've thought it just throws an error, otherwise I wouldn't wipe that. I probably rushed too much on that one 🙈 
Having another look, this whole file has a lot of TODOs and general uncertainty. For now I would remove `[NotImplemented]` and left the nurbs method as it is, as it is technically correct. And maybe raise an issue to sort the whole method out.

Reply:
as agreed in the call, lets just keep

Reply:
Yep - as per our Description in the oM - this is correct. So all looks good removing `[NotImplemented]`